microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -21,6 +21,8 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 | Language | Package | Source code | Notes |
 | --- | --- | --- | --- |
 | Node.js | [Edge Webstore Upload](https://www.npmjs.com/package/@plasmo-corp/ewu) | [GitHub](https://github.com/plasmo-corp/edge-webstore-upload)| This is a third party library. Microsoft makes no warranties, express or implied, with respect to the information provided here. |
+| .NET / C# | [EdgeAddonsAPI](https://www.nuget.org/packages/SantoshPisini.EdgeAddonsAPI) | [GitHub](https://github.com/SantoshPisini/edge-addons-api)| This is a third party library. Microsoft makes no warranties, express or implied, with respect to the information provided here. |
```

Comment:
move project to one of the Microsoft official GitHub organizations instead of /SantoshPisinini/
rendered:
https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3087#library-support